src/tests/ui/planner/workitem-list.ts

```diff
@@ -68,4 +69,15 @@ export class WorkItemList extends BaseElement {
   async isTitleTextBold(title: string) {
     return await this.workItem(title).title.getAttribute('className');
   }
+
+  async getUnassignedWorkItemCount(assigneeName: string) {
+    let assignee: any = await this.$$('f8-assignee').getAttribute('innerText');
+    let count = 0;
+    for (let i in assignee) {
```

Comment:
You could just do `new Set (assignee).size`

Reply:
@jarifibrahim why should i use new Set (assignee).size... I want to filter the result if assignee is unassigned and get the count... I changed from for loop to filter.